packages/ui-extensions/src/surfaces/admin/components/Card/Card.doc.ts

```diff
@@ -0,0 +1,42 @@
+import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
```

Comment:
I would rename this file to `Card.doc-skip.ts`. That way it doesn't get added to the next gen-docs command and we can settle on a mechanism to highlight gated components.